PhysicsTools/PatAlgos/plugins/PATMuonSlimmer.cc

```diff
@@ -101,10 +110,22 @@ pat::PATMuonSlimmer::produce(edm::Event & iEvent, const edm::EventSetup & iSetup
         }
     }
 
+    Handle<pat::PackedCandidateCollection> pc_h;
+    if(computeMiniIso_){
+        iEvent.getByToken(pc_, pc_h);
+    }
     for (vector<pat::Muon>::const_iterator it = src->begin(), ed = src->end(); it != ed; ++it) {
         out->push_back(*it);
         pat::Muon & mu = out->back();
         
+        if(computeMiniIso_){
+            pat::PFIsolation miniiso = pat::getMiniPFIsolation(pc_h.product(), mu.p4(),
+                                                                 miniIsoParams[0], miniIsoParams[1], miniIsoParams[2],
+                                                                 miniIsoParams[3], miniIsoParams[4], miniIsoParams[5],
+                                                                 miniIsoParams[6], miniIsoParams[7], miniIsoParams[8]);
+            mu.setMiniPFIsolation(miniiso);
+        }
+
         if( modifyMuon_ ) { muonModifier_->modify(mu); }
```

Comment:
the mini-iso computation depends on the object momentum
and the object itself here can be modified after in ``` muonModifier_->modify(mu); ``` 
this can lead to inconsistent computation.